thoth/storages/graph/postgres.py

```diff
@@ -6710,6 +6710,45 @@ def get_bloat_data(self) -> List[Dict[str, Any]]:
 
         return self._process_bloat_data_results(tables=tables)
 
+    def store_old_solver_documents(self, os_name=None, os_version=None, python_version=None):
```

Comment:
It might be a good idea to make these methods more generic - move them to the result base which would accept instantiated `GraphDabase` to query document ids and move them to a new location defined by a prefix (a CephStore can be instantiated within the implemented method with the specified prefix).